core/src/main/java/GameObjects/URLBuilder.java

```diff
@@ -0,0 +1,22 @@
+package GameObjects;
+
+/**
+ * The UrlBuilder class is a utility class used to build the URL for OpenDB required to retrieve
+ * questions given preferences regarding number of questions, category, and difficulty.
+ */
+public final class UrlBuilder implements UrlBuilderWrapper {
+  private final String base = "https://www.opentdb.com/api.php";
+  private final String difficultyBase = "&difficulty=";
+  private final String categoryBase = "&category=";
+  private final String amountBase = "?amount=";
+  private final String multipleChoice = "&type=multiple";
```

Comment:
Make these static final and make them uppercase with underscores (per the style guide and consistency with other constants).  I think 'MULTIPLE_CHOICE_QUERY' might be more descriptive for that constant.  I also think that the base should just be the query field (e.g. "difficulty"), and the ampersands and equals should be added in the buildUrl() method.